.github/workflows/linux.yml

```diff
@@ -0,0 +1,27 @@
+name: Linux Build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Build on Ubuntu
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@master
+    
+    - name: Setup Environment
+      run: |
+        sudo apt-get update
+        sudo apt-get install autoconf automake build-essential cmake git libass-dev libbz2-dev libfontconfig1-dev libfreetype6-dev libfribidi-dev libharfbuzz-dev libjansson-dev liblzma-dev libmp3lame-dev libnuma-dev 
+        sudo apt-get install libogg-dev libopus-dev libsamplerate-dev libspeex-dev libtheora-dev libtool libtool-bin libvorbis-dev libx264-dev libxml2-dev libvpx-dev m4 make nasm ninja-build patch pkg-config python tar yasm zlib1g-dev
+        sudo apt-get install python3-pip
+        sudo apt-get install python3-setuptools
+        sudo pip3 install meson
+        sudo apt-get install gstreamer1.0-libav intltool libappindicator-dev libdbus-glib-1-dev libglib2.0-dev libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev libgtk-3-dev libgudev-1.0-dev libnotify-dev
+        sudo apt-get install libwebkit2gtk-4.0-dev
+        sudo apt-get install libva-dev libdrm-dev
+        
+    - name: Build HandBrake Linux
+      run: |
+        ./configure --launch-jobs=0 --launch
```

Comment:
You might want to add --enable-qsv.  I can't really speak for vce and nvenc since I've not tried either on linux.  But at least qsv works well when combined with the right set of libs and drivers.

I would also add --disable-gtk-update-checks and dump libwebkit2gtk-4.0 dependency.  Update checks most likely don't work anymore anyway.  Been meaning to just remove that code.